modules/fedimint-mint-client/src/lib.rs

```diff
@@ -280,14 +285,19 @@ impl MintClientExt for Client {
             extra_meta: extra_meta.clone(),
         };
 
-        self.finalize_and_submit_transaction(
-            operation_id,
-            MintCommonGen::KIND.as_str(),
-            operation_meta_gen,
-            tx,
-        )
-        .await
-        .context("We already reissued these notes")?;
+        let change = self
+            .finalize_and_submit_transaction(
+                operation_id,
+                MintCommonGen::KIND.as_str(),
+                operation_meta_gen,
+                tx,
+            )
+            .await
+            .context("We already reissued these notes")?
+            .1;
+
+        self.await_primary_module_outputs(operation_id, change)
+            .await?;
```

Comment:
this made `reissue_external_notes` block forever when offline, is this expected behavior?

cc @elsirion @joschisan

Reply:
No it is not, the await_primary_module_outputs call should be in subscribe_reissue_external_notes. We need to return the change variable with the operation id and make it an argument in subscribe_reissue_external_notes.

Reply:
I thought it was just a copy paste bug, created an issue to discuss further
https://github.com/fedimint/fedimint/issues/4355